pump/storage/storage_test.go

```diff
@@ -272,20 +273,21 @@ func (as *AppendSuit) testWriteBinlogAndPullBack(c *check.C, prewriteValueSize i
 }
 
 func (as *AppendSuit) TestDoGCTS(c *check.C) {
+	failpoint.Enable("github.com/pingcap/tidb-binlog/pump/storage/l0-file-too-much", "return")
 	var value = make([]byte, 10)
 	append := newAppend(c)
 	defer cleanAppend(append)
 
 	var i int64
-	var n int64 = 1025 * 500
+	var n int64 = 1026 * 3214
```

Comment:
If the limit per batch is configurable, we can use some smaller value here so that the tests don't slow down.

Reply:
PTAL again @suzaku